x/posts/internal/types/msgs.go

```diff
@@ -24,11 +24,12 @@ type MsgCreatePost struct {
 	Creator        sdk.AccAddress    `json:"creator"`
 	CreationDate   time.Time         `json:"creation_date"`
 	Medias         PostMedias        `json:"post_medias,omitempty"`
```

Comment:
I think we can rename this to just `medias` 

```suggestion
	Medias         PostMedias        `json:"medias,omitempty"`
```